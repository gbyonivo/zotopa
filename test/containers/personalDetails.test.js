import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import combineReducers from '../../src/reducers';
import PersonalDetails from '../../src/containers/personalDetails';


const props = {
  fetchPersonalDetails: () => {}
};

const transactionReducer = {
  isFetchingPersonalDetails: false,
  errorFetchingPersonalDetails: 'I am just being lazy',
  personalDetails: {
    name: 'Ivo'
  }
};

describe('PersonalDetails Component', () => {
  it('should render correctly', () => {
    const actual = renderer.create(
      <Provider store={createStore(combineReducers({ transactionReducer }))}>
        <PersonalDetails {...props}/>
      </Provider>
    );
    expect(actual).toMatchSnapshot()
  });
});