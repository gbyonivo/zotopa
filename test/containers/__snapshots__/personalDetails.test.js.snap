// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalDetails Component should render correctly 1`] = `
<div
  className="personalDetails"
>
  <h2>
    Personal Account Details
  </h2>
  <div
    className="loading"
  />
</div>
`;
